internal/handlers/web/assets/app.js

```diff
@@ -3,6 +3,7 @@
   const selectedNumbers = [];
   initializeNumberGrid(selectedNumbers);
   setupGenerateButton(selectedNumbers);
+  fetchAppVersion();
```

Comment:
It would be better to call the `fetchAppVersion` function after setting up the Generate button. This way, if there's an error fetching the version number, it won't affect the main functionality of the page. [medium]